website_and_docs/content/documentation/webdriver/interactions/virtual_authenticator.zh-cn.md

```diff
@@ -31,7 +31,7 @@ Web 应用程序可以启用基于公钥的身份验证机制（称为 Web 身
 {{< badge-code >}}
 {{< /tab >}}
 {{< tab header="Python" >}}
-{{< badge-code >}}
+{{< gh-codeblock path="examples/python/tests/virtual_authenticator/virtual_authenticator_test.py#LL40C1-L40C1" >}}
```

Comment:
Do you pin the right lines?

And meanwhile, as you signed on the commit description, can you help to sync the part to all languages?
" Code example added (and I also added the example to all translated languages)"